libs/pxweb2-ui/src/lib/components/Tabs/Tabs.module.scss

```diff
@@ -0,0 +1,63 @@
+.tabs {
+  border-bottom: 1px solid var(--color-border-divider-subtle, #c3dcdc);
+  width: 100%;
+  align-items: flex-end;
+
+  &.fixed {
+    display: inline-flex;
+  }
+
+  &.scrollable {
+    display: flex;
+    padding-left: 20px;
+    overflow: auto;
+  }
+}
+
+.tab {
+  display: flex;
+  padding: 12px 16px;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+  position: relative;
+  border-radius: var(--border-radius-xsmall, 2px);
+  border: none;
+  cursor: pointer;
+  background-color: var(--px-color-background-default);
+
+  &:hover {
+    background: var(--color-surface-action-subtle-hover, #dbebeb);
+  }
+
+  &:active {
+    background: var(--color-surface-action-subtle-active, #c3dcdc);
+  }
+
+  &:focus-visible {
+    border: 3px solid var(--color-border-focus-boxshadow, #162327);
+    outline: 3px solid var(--color-border-focus-outline, #7e4ef9);
+    outline-offset: 3px;
+    z-index: 1;
+  }
+}
+
+.underline {
+  position: absolute;
+  bottom: 0px;
+  height: 4px;
+  border-radius: var(--border-radius-full, 9999px);
+  background: var(--color-border-selected, #274247);
+  width: 100%;
+  z-index: 1;
+}
+
+.label {
+  color: var(--color-text-default, #162327);
+  /* Label/Medium */
```

Comment:
I might remember incorrectly here, but don't we have a file we can import this text style from?